content/chainguard/libraries/javascript/build-configuration.md

```diff
@@ -216,36 +216,155 @@ To change the packages, remove the `node_modules` directory and the
 
 Now you can proceed with your development and testing. 
 
-<!--
 <a id="yarn"></a>
 
 ## Yarn
 
 [Yarn](https://yarnpkg.com/) is a popular package manager for JavaScript
 projects, offering fast, reliable, and secure dependency management as an
-alternative to npm. It is widely used for managing project dependencies,
-scripts, and workflows in Node.js and other JavaScript development environments.
-For more details, refer to the [Yarn
+alternative to npm. It is widely used for managing
+project dependencies, scripts, and workflows in Node.js and other JavaScript
+development environments. For more details, refer to the [Yarn
 documentation](https://yarnpkg.com/getting-started/documentation). 
 
 This section applies to modern versions of Yarn, also known as Yarn Berry, with
-versions 2.x and higher.
+versions 2.x and higher. If you are using Yarn 1.x refer to the [Yarn Classic
+section](#yarn-classic).
+
+Using yarn, dependencies to JavaScript packages are declared in a `package.json`
+file and separated into different scoped dependencies. development and runtime
+dependencies. The following block shows a minimal example with `react` and
+`node` as main runtime dependencies and `eslint` as development dependency:
 
+```json
+{
+  "name": "yarn-berry-example",
+  "packageManager": "yarn@4.10.3",
+  "dependencies": {
+    "node": "^22.20.0",
+    "react": "^19.1.1"
+  },
+  "devDependencies": {
+    "eslint": "^9.36.0"
+  }
+}
 ```
-yarn config set registry http://localhost:8081/repository/javascript-all/
+By default, the packages are retrieved from the registry at
```

Comment:
```suggestion
By default, `yarn` retrieves the packages from the registry at `https://registry.yarnpkg.com` and stores them locally in a `.yarn` folder in the user's home directory. 
```

Trying to get away from some of the passive voice in these updates

Reply:
so good.. updating the whole doc around that as well